package.yaml

```diff
@@ -52,6 +52,7 @@ library:
     - hspec-core
     - hspec-expectations-lifted
     - hspec-junit-formatter
+    - http-client
```

Comment:
Non-blocking: If we are using `http-client` simply to get at `HttpException`, is it possible to grab that from `http-conduit`? 

https://www.stackage.org/haddock/lts-17.10/http-conduit-2.3.8/Network-HTTP-Client-Conduit.html#t:HttpException

Reply:
You're right! Will change.

I really dislike how they've sliced up these packages...